src/averageAges.js

```diff
@@ -20,6 +20,17 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const mens = people.filter(({ sex }) => sex === 'm');
+
+  const sortedMens = century
+
+    ? mens.filter(({ died }) => Math.ceil(died / 100) === century)
+
+    : mens;
```

Comment:
I'm not really sure about these blank lines (same in all ternary operators throughout code)
```suggestion
  const sortedMens = century
    ? mens.filter(({ died }) => Math.ceil(died / 100) === century)
    : mens;
```